src/Container/AuthorizationRequestMiddlewareFactory.php

```diff
@@ -19,25 +19,24 @@
 namespace ZfrOAuth2\Server\Container;
 
 use Interop\Container\ContainerInterface;
-use ZfrOAuth2\Server\AuthorizationServerMiddleware;
 use ZfrOAuth2\Server\AuthorizationServer;
-use ZfrOAuth2\Server\Service\ClientService;
+use ZfrOAuth2\Server\Middleware\Endpoint\AuthorizationRequestMiddleware;
```

Comment:
I don't feel like the extra "Endpoint" in the namespace is necessary. You have few middlewrae, no need to break theme evne more.